bagitobjecttransfer/recordtransfer/forms/user_forms.py

```diff
@@ -60,7 +76,15 @@ def clean(self) -> "dict[str, Any]":
         current_password = cleaned_data.get("current_password")
         new_password = cleaned_data.get("new_password")
         confirm_new_password = cleaned_data.get("confirm_new_password")
-        gets_notification_emails = cleaned_data.get("gets_notification_emails")
+
+        first_name = cleaned_data.get("first_name")
+        last_name = cleaned_data.get("last_name")
+
+        if first_name and not self.NAME_PATTERN.match(first_name):
+            raise ValidationError({"first_name": [self.NAME_VALIDATION_MESSAGE]})
+
+        if last_name and not self.NAME_PATTERN.match(last_name):
+            raise ValidationError({"last_name": [self.NAME_VALIDATION_MESSAGE]})
```

Comment:
This is unnecessary with `RegexField`

```suggestion

        first_name = cleaned_data.get("first_name")
        last_name = cleaned_data.get("last_name")

        if first_name and not self.NAME_PATTERN.match(first_name):
            raise ValidationError({"first_name": [self.NAME_VALIDATION_MESSAGE]})

        if last_name and not self.NAME_PATTERN.match(last_name):
            raise ValidationError({"last_name": [self.NAME_VALIDATION_MESSAGE]})
```

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/281/commits/70fde46c390930b972d9f5174fa95b75d94f5ee7